csi/deployment.go

```diff
@@ -56,7 +57,7 @@ func NewAttacherDeployment(namespace, serviceAccount, attacherImage, rootDir str
 			"--leader-election-namespace=$(POD_NAMESPACE)",
 			"--leader-election-identity=$(POD_NAME)",
 		},
-		DefaultCSIDeploymentReplicaCount,
+		replicaCount,
```

Comment:
Convert to int32() here instead of accepting it as a parameter. `int32` is not commonly used. Same below.